apps/financial-aid/web-veita/src/components/Files/Files.tsx

```diff
@@ -3,7 +3,7 @@ import { Text, Box } from '@island.is/island-ui/core'
 
 import * as styles from './Files.treat'
 import cn from 'classnames'
-import { calcDifferenceInDate, getFileType } from '../../utils/formHelper'
+import { getFileType } from '../../utils/formHelper'
```

Comment:
@island.is/financial-aid-web/veita/src/utils/formHelper 
Maybe full path is better?